tests/fast-integration/utils/index.js

```diff
@@ -412,6 +413,38 @@ function waitForFunction(name, namespace = 'default', timeout = 90_000) {
   );
 }
 
+async function getSubscriptionConsumerName(subscriptionName, namespace='default', crdVersion='v1alpha1') {
+  const sub = await getSubscription(subscriptionName, namespace, crdVersion);
+  let consumerName;
+  if (crdVersion === 'v1alpha1') {
+    // the logic is temporary because consumer name is missing in the v1alpha1 subscription
```

Comment:
I did this hack because this was fast solution and temprorary as we in the near future upgrade to v1alpha2. Hope it is okay for you :)